src/api/SearchApi.js

```diff
@@ -76,38 +76,54 @@ class RequestSerializer {
 
 /** Default backend response serializer */
 class ResponseSerializer {
-  _serializeAggregation(bucket) {
-    const aggregation = {
-      key: bucket.key,
-      total: bucket.doc_count,
-      hasNestedField: false,
-    };
+  _serializeAggregations(aggregationsResponse) {
+      const createAggregation = (parentKeyName, buckets) => {
```

Comment:
I think split that to a `_ createAggregation` function would be more readable... WDYT?

Reply:
I agree, done!